javascript-modal/main.js

```diff
@@ -0,0 +1,27 @@
+var isOn = true;
+var modalButton = document.querySelector('.modal-button');
+var modalContainer = document.querySelector('.modal-container');
+
+function modalOpen() {
+  isOn = !isOn;
```

Comment:
Since you have this functionality in multiple different functions, a boolean flag is not needed. You can make use of a boolean flag if you only made one event handler function, but with two separate function, you can just do the work for the two separate situations without needing to check anything. If somebody clicks the "Open Modal" button, they want to open the modal every time. If somebody clicks the "No" button, they want to close the modal every time.